packages/neon/neon_news/lib/pages/article.dart

```diff
@@ -65,7 +65,7 @@ class _NewsArticlePageState extends State<NewsArticlePage> {
 
   Future _startMarkAsReadTimer() async {
     if (await widget.bloc.unread.first) {
-      if (widget.articlesBloc.options.articleDisableMarkAsReadTimeoutOption.value) {
+      if (widget.articlesBloc._options.articleDisableMarkAsReadTimeoutOption.value) {
```

Comment:
I don't like accessing private fields like that.
We should use a getter for this.

Maybe we should reconsider having our apps as a single library and also split them

Reply:
Oh that was a mistake, I tried hiding the options where it is possible.